qermod/types.py

```diff
@@ -37,5 +37,5 @@ class NoiseCategory:
     """Noise applied to digital blocks."""
 
 
-NoiseCategoryEnum = Union[DigitalNoise, AnalogNoise, ReadoutNoise]
+NoiseEnum = Union[DigitalNoise, AnalogNoise, ReadoutNoise]
```

Comment:
Can't you simply do smtg like:

```python
class DigitalNoise(StrEnum):
     BITLFIP
     ...

class AnalogNoise(StrEnum):
    ...

class ReadoutNoise(StrEnum):
   ...

class Noise(StrEnum):
    DIGITAL = DigitalNoise
    ...
```

So that everything gets gathered into a single `Noise` type ?